_posts/2020-04-03-rendering-in-UE4.md

```diff
@@ -0,0 +1,32 @@
+---
+author: "alepaoletti"
+layout: post
+did: "blog10"
+title: "Rendering in Unreal Engine 4"
+slug: "RenderingInUE4"
+date: 2020-04-06 08:00:00
+categories: rendering
+img: /banners/GPU_profiler.png
+banner: /banners/GPU_profiler.png
+tags: graphics
+description: "A really quick overview of the rendering pipeline in Unreal Engine 4."
+---
+
+Unreal Engine 4 has a lot of code for doing rendering: it exists in its own module - the *Renderer* module - which is compiled to a dll to allow faster iteration. The *Renderer* module depends on the Engine module because it has many callbacks into Engine. However, when the Engine needs to call some code in the Renderer, this happens through the *IRendererModule* interface.
```

Comment:
IMHO you can change compiled to a DLL to "compiled to a shared library (DLL on Windows, .so on Linux, .dylib in MacOS, ...)" or something similar, since ue4 is cross-platform